src/pages/resultsView/comparison/ResultsViewComparisonStore.ts

```diff
@@ -18,6 +18,7 @@ import ifNotDefined from '../../../shared/lib/ifNotDefined';
 import comparisonClient from '../../../shared/api/comparisonGroupClientInstance';
 import { ComparisonGroup } from '../../groupComparison/GroupComparisonUtils';
 import { ComparisonSession } from 'shared/api/session-service/sessionServiceModels';
+import { LEFT_TRUNCATION_ADJUSTMENT_FLAG } from '../survival/SurvivalUtil';
```

Comment:
lets keep these feature flag constants in /shared/featureFlags.ts

Reply:
Comments addressed in https://github.com/cBioPortal/cbioportal-frontend/pull/4324/commits/1359463a4b53e54a0530bdb509f659ebb929368b